includes/allDocsGenerator.js

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2023 IBM Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict';
+
+const allDocsGenDebug = require('debug')('couchbackup:alldocsgenerator');
```

Comment:
can we change this to `debug` for consistency with other modules?

Reply:
There's a bit of inconsistency as we currently have 7 places that do `require('debug')` and 6 places (including this one) that do `require('debug')(...)`.  I can change it to `debug` but we should decide on one or the other before merging the modernization work into main.

Reply:
I, sorry, I wasn't clear. I mean lets do `const debug = require('debug')('couchbackup:alldocsgenerator');` as everywhere else, that `allDocsGenDebug` is mouthful

Reply:
Ah ok good call, updated in [9beaace](https://github.com/IBM/couchbackup/pull/673/commits/9beaace6de141d1caf1a186a7961bff644c728d7)